test/integration/targets/rabbitmq_lookup/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+- import_tasks: tests.yml
+  when: ansible_distribution == 'Ubuntu'
```

Comment:
What about the other available distributions ? Add a comment explaining why there are not tested.

Reply:
I've removed this line now.  Next push for update.